crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```diff
@@ -22,6 +22,7 @@ url2 = "0.0.6"
 serde = { version = "1.0.104", features = [ "derive" ] }
 serde_bytes = "0.11"
 fixt = { path = "../../fixt" }
+holo_hash = { path = "../../holo_hash" }
```

Comment:
No, holo_hash is specific to holochain. If Kitsune is to be extractable it cannot depend on it.